src/config.py

```diff
@@ -34,6 +34,22 @@
 LAYOUTPARSER_MIN_OVERLAPPING_PIXELS_VERTICAL = int(
     os.getenv("LAYOUTPARSER_MIN_OVERLAPPING_PIXELS_VERTICAL", "5")
 )
+# Percentage of page to ignore at the top of the page when adding blocks to the
```

Comment:
Good level of comments should they sit in a readme? 

Alternatively group into a config object and put the comments there:


```
class LayoutParserConfig(BaseModel):
   """Config for the layout parser""" 
   top_exclude_threshold: float 
   .... 
```

Reply:
Fair comment. For now I'll just put a pointer in readme instead of editing as refactoring this really not that important.